Solita.LocalizationEditor.UI/DAL/FileAccessStrategy.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using System.Xml;
+using EPiServer.Framework.Blobs;
+using Solita.LocalizationEditor.UI.Models;
+
+namespace Solita.LocalizationEditor.UI.DAL
+{
+    public abstract class FileAccessStrategy
+    {
+        public string FilePath { get; }        
```

Comment:
It looks that FilePath property is never used. By the way, did you consider using interface instead of abstract class for FileAccessStrategy?